airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/spec.json

```diff
@@ -50,6 +50,25 @@
                 "airbyte_secret": true
               }
             }
+          },
+          {
+            "type": "object",
+            "title": "Authenticate via Lever (Api Key)",
+            "required": ["api_key"],
+            "properties": {
+              "auth_type": {
+                "type": "string",
+                "const": "Api Key",
+                "enum": ["Api Key"],
```

Comment:
I think an api key is dynamic for each develop application. is it a string enum? Essentially, the spec.json comes from JSON Schema, and [here](https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values) is the doc for enum.